Feature modules and styles to development

SofiBretz opened the pull request:
In this project:

- Created 2 files for the project
- Added 'Webpack' and 'eslint'.
- Added Style file to the app.

MauricioRobayo commented on .stickler.yml:
```diff
@@ -0,0 +1,16 @@
+files:
+  ignore:
+      - 'dist/*'
+      - 'webpack.config.js' 
```
Please do not modify the provided configuration files: https://github.com/microverseinc/linters-config/blob/master/javascript/.stickler.yml.

MauricioRobayo commented on README.md:
```diff
@@ -1 +1,62 @@
+
+## Live Demo
+
+[Live Demo Link]()
```
Don't forget about the live demo link.

MauricioRobayo commented on README.md:
```diff
@@ -1 +1,62 @@
+
+### Install
+
+- Install npm and webpack commands.
```
Webpack and family should be listed on your devDependencies, so it should be install when you run `npm install` to install.

Oghenebrume50 replied:
Please see this

SofiBretz replied:
fixed

MauricioRobayo commented on dist/index.html:
```diff
@@ -0,0 +1,87 @@
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="../css/main.css">
```
You could load this with Webpack on `index.js`, you even have the dependencies to do it on your package.json.

MauricioRobayo commented on package.json:
```diff
@@ -0,0 +1,51 @@
+    "jquery": "^3.4.1",
+    "popper.js": "^1.16.1",
+    "webpack": "^4.42.0",
+    "webpack-dev-server": "^3.10.3"
```
This should be devDependencies as you are using those to build your project, they are not going to be needed on production.

Oghenebrume50 replied:
Hi please see this

SofiBretz replied:
fixed

MauricioRobayo commented on package.json:
```diff
@@ -0,0 +1,51 @@
+  "dependencies": {
+    "bootstrap": "^4.4.1",
+    "jquery": "^3.4.1",
+    "popper.js": "^1.16.1",
```
Are you using Bootstrap on this project? 🤔

MauricioRobayo commented on src/index.js:
```diff
@@ -0,0 +1,142 @@
+const save = () => {
+  localStorage.setItem(LOCAL_STORAGE_LIST_KEY, JSON.stringify(lists));
+  localStorage.setItem(LOCAL_STORAGE_SELECTED_LIST_ID_KEY, selectedListId);
+};
```
Maybe you could refactor your localStorage logic into a separate module.

MauricioRobayo commented on webpack.config.js:
```diff
@@ -0,0 +1,36 @@
+        },
+      },
+      {
+        test: /\.css$/,
```
You could use the sass-loader that you have on your devDeps for your `scss` files and let webpack do its magic.

SofiBretz commented:
### Summary of Changes:

- Add the requirements on the readme file.
- About the changes on the stickler, I ignore the webpack file because when I made the changes with stickler it asks me to install some package when I did it it brokes my code, so I thought it was not so necessary to not ignore this request on the eslint.
- The user can 'delete' the tasks and the list:
 
![2](https://user-images.githubusercontent.com/42303392/77493670-564efe80-6e09-11ea-8207-7570e7ff9a0e.png)

-There is a 'default' task on the app. On line 32 and 33.

![Screenshot from 2020-03-24 19-55-42](https://user-images.githubusercontent.com/42303392/77493785-9f9f4e00-6e09-11ea-8391-151a6f37a45e.jpg)

![1](https://user-images.githubusercontent.com/42303392/77493719-6961ce80-6e09-11ea-908e-d1074f0a0e08.png)

Thanks for your advice. Ready for a code review.

SofiBretz commented:
### Summary of Changes:

- I saw that the 'default' task was presenting a problem, I fixed now. I apologize for that, in my pc was working. Now I fixed the bug. :)

![11](https://user-images.githubusercontent.com/42303392/77560293-e41b0000-6e82-11ea-9af2-d9564ad222ba.png)

-The dependencies are fixed, now the advice of the last TSE is done, thank you!.

- As I explain with the image to the first TSE, you can delete a 'task' and the 'to-do list' also, if you click on the 'Clear completed tasks' button is going to delete the task that you already marked as a complete, and if you click the 'Delete list' button is going to delete the todo list. So it is not a problem with this since the beginning, maybe you miss to try these buttons. I attach 3 images of the steps so you can double-check that I'm doing the requirements properly.

![22](https://user-images.githubusercontent.com/42303392/77560341-f6953980-6e82-11ea-90c5-85ff47d3c0f6.png)

![222](https://user-images.githubusercontent.com/42303392/77560322-eda46800-6e82-11ea-8b39-5f6024aec63b.png)

![122](https://user-images.githubusercontent.com/42303392/77560340-f5640c80-6e82-11ea-8380-662f7109a868.png)

All the requirements of the 2 TSE's fixed. Ready for a code review. :+1:

azdrenymeri commented:
### Hello @SofiBretz :wave: 
### All the requirements seems to be filled  :heavy_check_mark: 
### No further requirements, you can merge :+1:
